docs/docs/self-host/env.mdx

```diff
@@ -155,7 +155,7 @@ object with the settings will need to be JSON encoded and then set to the enviro
 - **SSO_CONFIG** - A JSON-encoded string that configures SSO (using OpenID Connect). It should be an object with the following keys:
   - `clientId` (string)
   - `clientSecret` (string)
-  - `emailDomain` (string, optional) - Allow auto-joining from a specified email domain
+  - `emailDomains` (string[], optional) - Allow auto-joining from only specified email domains
```

Comment:
There are references to `emailDomain` 4 other times in this file we should fix as well